.env

```diff
@@ -3,5 +3,6 @@ IMAGE_TAG=develop
 ENVIRONMENT=production
 DESCRIPTION=
 TEST_HTML=
+NODE_VERSION=6.14.3
```

Comment:
This makes my eyes bleed. Update of this to LTS coming soon (as a separate PR).

Reply:
Expires in 2019 fortunately